packages/contact-manager-rest-api/__tests__/agent.ts

```diff
@@ -14,48 +21,70 @@ const dbConnection = DataSources.singleInstance()
 const agent = createAgent<IRequiredPlugins>({
   plugins: [new DataStore(dbConnection), new DataStoreORM(dbConnection), new ContactManager({ store: new ContactStore(dbConnection) })],
 })
+const generateIdentity = (contact: Record<string, any>) : NonPersistedIdentity => {
```

Comment:
Please use the agent method to generate a key and never use PII in DIDs (even though this is not stored on a ledger, never use PII for them)